tests/hashtree.nim

```diff
@@ -0,0 +1,6 @@
+import ../hashtree_abi
+
+var data: array[64, byte]
+hashtree_hash(addr data[0], addr data[0], 1)
+
+doAssert data[0] == 245
```

Comment:
quite minimal, but tests the core funtionality of the binding. more exhaustive tests should probably not be nim specific, so is ok